Tests/Marketplace/search_and_install_packs.py

```diff
@@ -687,6 +687,8 @@ def search_and_install_packs_and_their_dependencies(pack_ids: list,
     installation_request_body: list = []  # the packs to install, in the request format
     batch_packs_install_request_body: list = []    # list of lists of packs to install if install packs one by one.
     # Each list contain one pack and its dependencies.
+    packs_to_install_together: list = []
+    installed_packs: list = []
```

Comment:
Let's use `batch_packs_install_request_body` as a batch list instead of `packs_to_install_together` with CUSTOM_BATCH_SIZE.  For XSIAM  CUSTOM_BATCH_SIZE=1, for NG=CUSTOM_BATCH_SIZE. 

And change `install_packs_one_by_one` to `install_packs_custom_batch_size`.